src/frame/js/components/ChatInput.jsx

```diff
@@ -132,10 +132,10 @@ export class ChatInputComponent extends Component {
     }
 }
 
-export default connect(({appState, app, ui, conversation: {unreadCount}}) => {
+export default connect(({ui, conversation: {unreadCount}, config}) => {
     return {
-        imageUploadEnabled: appState.imageUploadEnabled,
-        accentColor: app.settings.web.accentColor,
+        imageUploadEnabled: config.imageUploadEnabled,
```

Comment:
(a bit lost here, sorry) Is this config the same as the response from `/config`? If so, where is `imageUploadEnabled` assigned to it?